editor/filesystem_dock.cpp

```diff
@@ -228,10 +228,10 @@ void FileSystemDock::_update_tree(const Vector<String> &p_uncollapsed_paths, boo
 	DirAccessRef da = DirAccess::create(DirAccess::ACCESS_RESOURCES);
 	bool fav_changed = false;
 	for (int i = favorite_paths.size() - 1; i >= 0; i--) {
-		if (!da->dir_exists(favorite_paths[i])) {
-			favorite_paths.remove_at(i);
-			fav_changed = true;
-		}
+		if (da->dir_exists(favorite_paths[i]) || da->file_exists(favorite_paths[i]))
+			continue;
```

Comment:
```suggestion
		if (da->dir_exists(favorite_paths[i]) || da->file_exists(favorite_paths[i])) {
			continue;
		}
```
Single-line `if`s also use brackets in Godot.

Reply:
It is adjusted now, thank you.